src/__integrationtests__/jest.teardown.js

```diff
@@ -0,0 +1,7 @@
+const { getCached } = require('../blockchainApiConnection')
+
+module.exports = function teardown() {
+  return getCached()
+    .then((bc) => bc.api.disconnect())
+    .finally(process.exit())
```

Comment:
Do we really need to exit here?

Reply:
I reverted the tear down in 2fdb8dc. It only made sense for the integration tests and since you cannot add a teardown via CLI or run it for a subset of tests (AFAIK).